src/native/libs/System.Globalization.Native/pal_casing.m

```diff
@@ -149,4 +149,25 @@ int32_t GlobalizationNative_ChangeCaseInvariantNative(const uint16_t* lpSrc, int
     }
 }
 
+void GlobalizationNative_InitOrdinalCasingPageNative(int32_t pageNumber, UChar* pTarget)
```

Comment:
Now it does not get trimmed in hybrid mode. But once we will not load ICU at all `pal_casing.c` will also not be loaded.

Reply:
> But once we will not load ICU at all pal_casing.c will also not be loaded.

What would this look like? Would it get trimmed out or would it be a file exclusion?
Since this and `GlobalizationNative_InitOrdinalCasingPage` have the exact same logic`, it would be nice if we found a way to not duplicate this code.

Reply:
From here https://github.com/dotnet/runtime/blob/f2e4ddf20936b37445bbbb2d139e8080ca00f16c/src/mono/mono/mini/CMakeLists.txt#L61 we will filter out for Apple platforms `pal_casing.c`

Reply:
Our goal is to make hybrid globalization default mode for Apple platforms and in that case we will not need most of the  ICU related functions (we will not need most of the `pal_"name".c`). I guess at the end to not have duplicates I can move all these kind of functions in some common place like `pal_common.c`

Reply:
No this was first function. Moved it to pal_common.c .